engine/src/test/scala/cromwell/engine/backend/mock/DefaultBackendJobExecutionActor.scala

```diff
@@ -19,7 +19,7 @@ case class DefaultBackendJobExecutionActor(override val jobDescriptor: BackendJo
   }
   override def recover = execute
 
-  override def abortJob: Unit = ()
+  override def abort: Unit = ()
```

Comment:
ToL: I can't remember why this is `Unit` instead of `Future[Unit]`... I assume that this doesn't block any actors' `receive` methods if we have a slow-acting `abort` method?